pkg/framework/test/examples_api_server_forced_args_process_starter_test.go

```diff
@@ -0,0 +1,32 @@
+package test_test
+
+import (
+	"io"
+	"os/exec"
+
+	"fmt"
+
+	"github.com/onsi/gomega/gexec"
+	. "k8s.io/kubectl/pkg/framework/test"
+)
+
+func ExampleAPIServer_forcedArgsProcessStarter() {
+	apiServer := &APIServer{
+		ProcessStarter: forceArgsStarter,
+	}
+	fmt.Println(apiServer)
+}
+
+func forceArgsStarter(command *exec.Cmd, out, err io.Writer) (SimpleSession, error) {
```

Comment:
That seems like a valid feature, but I have the feeling that:
- Asking users to run `gexec.Start` might be too much,
- Exposing the `SimpleSession` shouldn't be necessary

Beside arguments, do we expect them to change anything else?